src/main/java/javax/security/jacc/PolicyConfiguration.java

```diff
@@ -20,568 +20,408 @@
- *        <td width="12%" align="center">
- *          <font size="-2">inService</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">inService</font></td>
- *     </tr>
- *     <tr>
- *        <td width="28%"><font size="-2">delete</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">deleted</font></td>
- *        <td width="12%" align="center">
- *          <font size="-2">deleted</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">deleted</font></td>
- *     </tr>
- *     <tr>
- *        <td width="28%"><font size="-2">getContextID</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">deleted</font></td>
- *        <td width="12%" align="center">
- *          <font size="-2">open</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">inService</font></td>
- *     </tr>
- *     <tr>
- *        <td width="28%"><font size="-2">inService</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">deleted</font></td>
- *        <td width="12%" align="center">
- *          <font size="-2">open</font></td>
- *        <td width="25%" align="center">
- *          <font size="-2">inService</font></td>
- *     </tr>
- *     <tr>
- *       <td width="28%"><font size="-2">linkConfiguration</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *       <td width="12%" align="center">
- *         <font size="-2">open</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *     </tr>
- *     <tr>
- *       <td width="28%"><font size="-2">removeExcludedPolicy</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *       <td width="12%" align="center"><font size="-2">
- *         open</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *     </tr>
- *     <tr>
- *       <td width="28%"><font size="-2">removeRole</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *       <td width="12%" align="center">
- *         <font size="-2">open</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *     </tr>
- *     <tr>
- *       <td width="28%"><font size="-2">removeUncheckedPolicy</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *       <td width="12%" align="center">
- *         <font size="-2">open</font></td>
- *       <td width="25%" align="center">
- *         <font size="-2">Unsupported Operation</font></td>
- *     </tr>
+ * <tr>
+ * <th valign="middle" rowspan="2" colspan="1" >Method</th>
```

Comment:
I removed them primarily because the Javadoc tool emitted a lot of warnings. We could replace them by CSS, but maybe it's not the job of Javadoc to specify styling. I'm not sure.

Reply:
Sorry, I meant HTML tabulation. Removing the widths is good as it was redundant (only the first row needs to set the width) and it was difficult to read.

Tabulation makes it a bit easier to read but more difficult to maintain, since IDE formatters usually don't respect it. Let's leave it as it is.